backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/job/service/commands/PickingJobCreateCommand.java

```diff
@@ -136,6 +143,36 @@ private PickingJob executeInTrx()
 		}
 	}
 
+	@NonNull
+	private PickingJob allocatePickingSlotIfPossible(@NonNull final PickingJob pickingJob)
+	{
+		final PickingSlotId pickingSlotId = loadingSupportServices.getWorkplaceByUserId(request.getPickerId())
```

Comment:
why we introduced getWorkplaceByUserId to loading support service
and why not keeping that logic here?

asking because IMHO getWorkplaceByUserId is not related to "loading services"